src/components/policy/policy_external/test/sql_pt_representation_test.cc

```diff
@@ -1682,11 +1682,12 @@ TEST_F(SQLPTRepresentationTest,
   table["policy_table"]["module_meta"] = Json::Value(Json::objectValue);
   table["policy_table"]["module_config"]["preloaded_pt"] = Json::Value(false);
   policy_table::Table expected(&table);
-  Json::StyledWriter writer;
+  Json::StreamWriterBuilder writer_builder;
   // Checks
-  EXPECT_EQ(writer.write(expected.ToJsonValue()),
-            writer.write(snapshot->ToJsonValue()));
-  std::cout << writer.write(snapshot->ToJsonValue()) << std::endl;
+  EXPECT_EQ(Json::writeString(writer_builder, expected.ToJsonValue()),
+            Json::writeString(writer_builder, snapshot->ToJsonValue()));
+  std::cout << Json::writeString(writer_builder, snapshot->ToJsonValue())
```

Comment:
i know its a test, but we should save `snapshot->ToJsonValue()` instead of calculating it twice

Reply:
done in [2d489f0](https://github.com/smartdevicelink/sdl_core/pull/3155/commits/2d489f052063a2b121cd3a78ba26e2bad2fccc29)